### PR TITLE
Redis password

### PR DESCRIPTION
**Pull request by HellScre4m**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY
Added support for external Redis password and db number when deploying via docker.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: contrib/docker/superset_config.py [hunk 1]
```diff
@@ -45,12 +45,18 @@ def get_env_variable(var_name, default=None):
...
 REDIS_PORT = get_env_variable('REDIS_PORT')
-
+REDIS_PASSWORD = get_env_variable('REDIS_PASSWORD')
+REDIS_DBNUMBER = get_env_variable('REDIS_DBNUMBER')
 
 class CeleryConfig(object):
     BROKER_URL = 'redis://%s:%s/0' % (REDIS_HOST, REDIS_PORT)
     CELERY_IMPORTS = ('superset.sql_lab', )
-    CELERY_RESULT_BACKEND = 'redis://%s:%s/1' % (REDIS_HOST, REDIS_PORT)
+    if REDIS_DBNUMBER == '' or REDIS_DBNUMBER is None:
```
Comment (mistercrunch) [inline contrib/docker/superset_config.py]: Logic (`if`) in class definition seems like a bad idea. How about setting `CELERY_RESULT_BACKEND` above in module scope?

**Review by mistercrunch — CHANGES_REQUESTED** [review]

**Comment by syazwan0913** [comment]
@mistercrunch Is there any plan to implement Redis password in current Superset?

**Comment by mistercrunch** [comment]
@syazwan0913 you can just put it in your connection string can't you?

**Comment by syazwan0913** [comment]
@mistercrunch Ooo yeah i didn't think of that. Will try it out. Thanks

**Comment by syazwan0913** [comment]
@mistercrunch but even put into connection string, need to do 'requirepass' directive in the redis.conf configuration file right?

**Comment by mistercrunch** [comment]
Mmmh idk, seems like a Redis-related question.

**Comment by syazwan0913** [comment]
@mistercrunch It's okay already got it.

**Pull request closed without merging**